thinkhazard_processing/models.py

```diff
@@ -43,6 +45,11 @@ class Dataset(Base):
     # it maintains the application state
     processed = Column(Boolean, default=False)
 
+    def path(self):
+        return os.path.join(settings['data_path'],
+                            'datasets',
+                            self.hazard_set_id)
+
```

Comment:
A `path` for a dataset doesn't really mean anything to me right now. Is it really useful? If not, let's remove it.